### ISSUE TITLE
Could you please put pretrained weights to huggingface/mega.nz/google drive, or any other cloud outside chine? Thanks

### ISSUE THREAD

**Issue by zelenooki87** [QMoQ/OAPT#1]
Thank you for wonderful work.
Unfortunately I can not download models from pan baidu. Could you upload them somewhere else?
Thanks

**Comment by Arcitec** [comment]
Yes, unfortunately putting the files on Baidu Netdisk is basically the same thing as not sharing the files at all.

Baidu Netdisk:

- Requires Chinese phone number to sign up.
- Requires a Baidu NetDisk phone app that cannot be installed on western Google Play Store.
- Requires alternatively a binary app on the computer which I cannot verify the safety of. And if I could do it, I still can't make an account because I don't have a Chinese phone number.
- There are no download services or scripts or alternative GitHub repos for downloading from Baidu because they really crush down on those services.

Please @QMoQ, put it on HuggingFace, it's the world's AI playground! It's free and it works. :)

**Comment by Eric-Hao** [comment]
Hi, @zelenooki87 @Arcitec 

Thanks for your advice. All model weights are now available for download from [Gdrive](https://drive.google.com/drive/folders/1yZcczbsVdxmsocQMaC9oFcvt6J553D_1?usp=sharing).

cc @QMoQ
